docs/usage/faq.md

```diff
@@ -0,0 +1,49 @@
+## How does tuist overlap with SPM?
+
+Swift Package Manager (SPM) is the official tool from Apple for managing Swift packages. Although spm and tuist have a similar design, their goal is different. The Swift Package Manager aims to facilitate the definition and usage of Swift packages, while tuist facilitates the definition of projects _(e.g. iOS apps projects)_ and the interactions with them. Developers continue to ask Apple for some integration with existing Xcode projects so you might expect some overlapping in the future.
+
+## Can I use it with Fastlane?
+
+Definitively. Although tuist provides CLI commands such as a build and test, you can use Fastlane's gym and scan actions, or your own automation logic. You can work with the project that tuist generates as if it was a project generated by Xcode, nothing changes.
+
+## Why are the manifests in Swift?
+
+First and foremost, because the manifest files like `Project.swift` or `Workspace.swift` can be edited with Xcode, which helps the developers validate the syntax using the Swift type system. It also prevents developers from tweaking tuist in any way. This might change in the future, and there'll be support for hooks where developers will have the flexibility to define custom actions.
+
+## Can my targets depend on external dependencies?
+
+They can, but there's no dependency management built in tuist. Depending on how your project manages external dependencies, the integration process changes:
+
+- [Carthage](https://github.com/carthage/carthage): Since it generates precompiled framework, you can set your target up to depend on that framework. tuist will set up the build settings and build phases for you.
+- [CocoaPods](https://cocoapods.org): The project manifest model doesn't support adding a CocoaPods dependency but you can call `pod install` after the project has been generated with tuist. We have plans to improve this having support for it from the manifest.
+- [Swift Package Manager](https://swift.org/package-manager/): Unfortunately, there's no support for integrating your projects with spm dependencies. We are waiting for Apple to do some work on that area to add the integration into tuist.
+
+{% include faq-question.html question="Should I check the Xcode projects into source control?" %}
+Whether or not you check in your projects is up to you, as workflows vary from project to project. We recommend that you keep the projects under source control, and don't add it to your .gitignore. But ultimately this decision is up to you:
+
+**Benefits of checking in the projects**
+
+- After cloning the repo, the project can immediately build and run, even without having Tuist installed on the machine.
+- The projects are guaranteed to be identical to those in the original installation after cloning the repo.
+
+**Benefits of ignoring the projects**
+
+- There won't be any conflicts to deal with when performing source control operations, such as merging branches with different Pod versions.
```

Comment:
If we prefer people checking in projects we could show them how to setup a git attribute merge rule to regenerate projects on conflicts to automate this problem.

see `Defining a custom merge driver`: https://git-scm.com/docs/gitattributes

This is not for now, but future consideration